# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1465009516 25200
#      Fri Jun 03 20:05:16 2016 -0700
# Node ID 16bb4651a54d97cdc0fd3e00891dae94bc258f45
# Parent  96c56993e679ca41547711f2e452028ccaff0a89
maint: Include profiler HTML templates in the distribution (bug #48116)

* scripts/profiler/module.mk (scripts_EXTRA_DIST): Include
$(scripts_profiler_html_DATA) in the list.

diff --git a/scripts/profiler/module.mk b/scripts/profiler/module.mk
--- a/scripts/profiler/module.mk
+++ b/scripts/profiler/module.mk
@@ -20,8 +20,11 @@ scripts_profiler_html_DATA = \
   scripts/profiler/html/style.css
 
 FCN_FILES += \
   $(scripts_profiler_FCN_FILES)
 
 PKG_ADD_FILES += scripts/profiler/PKG_ADD
 
 DIRSTAMP_FILES += scripts/profiler/$(octave_dirstamp)
+
+scripts_EXTRA_DIST += \
+  $(scripts_profiler_html_DATA)
